pokemongo_bot/cell_workers/pokemon_catch_worker.py

```diff
@@ -405,8 +405,7 @@ def _do_catch(self, pokemon, encounter_id, catch_rate_by_ball, is_vip=False):
                 )
 
                 # We could refresh here too, but adding 3 saves a inventory request
-                candy = inventory.candies().get(pokemon.num)
-                candy.add(3)
+                candy = inventory.candies(True).get(pokemon.num)
```

Comment:
Refreshing is a bad behaviour. It would be great if we could make `PokemonCatchWorker` update the candy account using the inventory variable when pokemon is captured is avoid this refresh here.